tests/page-objects/common/common.po.js

```diff
@@ -113,6 +113,9 @@ module.exports = {
     helper.waitElementToBeVisible(element(by.id('reports-list')));
     if (refresh) {
       browser.refresh();
+    } else {
+      element(by.css('.reset-filter')).click();
+      helper.waitForAngularComplete();
```

Comment:
A trick to trigger a new search. When already on the "reports" page, click on the menu item to "go to reports" doesn't, in fact, do anything.

Reply:
Done.